test/conformance/ingressclass/feature.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ingressclass
+
+import (
+	"fmt"
+
+	"github.com/cucumber/godog"
+	"github.com/cucumber/messages-go/v10"
+
+	"sigs.k8s.io/ingress-controller-conformance/test/kubernetes"
+	tstate "sigs.k8s.io/ingress-controller-conformance/test/state"
+)
+
+var (
+	state *tstate.Scenario
+)
+
+// IMPORTANT: Steps definitions are generated and should not be modified
+// by hand but rather through make codegen. DO NOT EDIT.
+
+// InitializeScenario configures the Feature to test
+func InitializeScenario(ctx *godog.ScenarioContext) {
+	ctx.Step(`^an Ingress resource in a new random namespace$`, anIngressResourceInANewRandomNamespace)
+	ctx.Step(`^The Ingress status should not show the IP address or FQDN$`, theIngressStatusShouldNotShowTheIPAddressOrFQDN)
+
+	ctx.BeforeScenario(func(*godog.Scenario) {
+		state = tstate.New()
+	})
+
+	ctx.AfterScenario(func(*messages.Pickle, error) {
+		// delete namespace an all the content
+		_ = kubernetes.DeleteNamespace(kubernetes.KubeClient, state.Namespace)
+	})
+}
+
+func anIngressResourceInANewRandomNamespace(spec *messages.PickleStepArgument_PickleDocString) error {
+	ns, err := kubernetes.NewNamespace(kubernetes.KubeClient)
+	if err != nil {
+		return err
+	}
+
+	state.Namespace = ns
+
+	ingress, err := kubernetes.IngressFromManifest(state.Namespace, spec.GetContent())
+	if err != nil {
+		return err
+	}
+
+	err = kubernetes.DeploymentsFromIngress(kubernetes.KubeClient, ingress)
+	if err != nil {
+		return err
+	}
+
+	err = kubernetes.NewIngress(kubernetes.KubeClient, state.Namespace, ingress)
+	if err != nil {
+		return err
+	}
+
+	state.IngressName = ingress.GetName()
+
+	return nil
+}
+
+func theIngressStatusShouldNotShowTheIPAddressOrFQDN() error {
+	_, err := kubernetes.WaitForIngressAddress(kubernetes.KubeClient, state.Namespace, state.IngressName)
```

Comment:
I'm a bit concerned about having to wait 5 minutes for this test to pass... I understand why we need to wait in regular tests, and can't assume this test scenario would pass by checking only once on the Ingress status. Can we override the default `kubernetes.WaitForIngressAddressTimeout` in this case?

Reply:
Yes, here https://github.com/kubernetes-sigs/ingress-controller-conformance/blob/master/conformance_test.go#L62